tmt/schemas/provision/hardware.yml

```diff
@@ -0,0 +1,220 @@
+---
```

Comment:
@happz seems not rebased? this file should not be here

Reply:
Should be fixed, the history was pretty weird.